LocasticTcomPayWayPayumBundle.php

```diff
@@ -18,7 +18,7 @@ public function build(ContainerBuilder $container)
         /** @var $extension PayumExtension */
         $extension = $container->getExtension('payum');
 
-        $extension->addPaymentFactory(new TcomOffsitePaymentFactory());
-        $extension->addPaymentFactory(new TcomOnsitePaymentFactory());
+        $extension->addGatewayFactory(new TcomOffsiteGatewayFactory());
+        $extension->addGatewayFactory(new TcomOnsiteGatewayFactory());
```

Comment:
FYI: PayumBundle 2.0 does not use these factories any more, the payum's factories (one extends Payum\Core\GatewayFactoryInterface) is used.